package.json

```diff
@@ -93,6 +94,7 @@
     "core-js": "^3.0.0",
     "dompurify": "^2.1.1",
     "hot-formula-parser": "^4.0.0",
+    "hyperformula": "0.5.0",
```

Comment:
For now I'd put `hyperformula` in `devDependencies` (and eventually in `dev-` and `peer-` dependencies, I guess?)

Reply:
🤷 @wojciechczerniak

Reply:
Should be `dev-` for tests. But probably `peer-` and `optional-` for production. Unless we decide to distribute the plugin in a separate package. Call is scheduled for next friday

Reply:
At the moment of writing this branch still technically uses it as a `dependency`, #7823 changes the initialization to allow us to move it out, resolving this as #7823 will move it there anyway.